getSampleData.js

```diff
@@ -0,0 +1,19 @@
+function getSampleData() {
+  const sampleData = `
+Timestamp,Address,ZIP,FullName,FooDuration,BarDuration,TotalDuration,Notes
```

Comment:
This add a blank line at the top of the data

Reply:
I ran into this also, and I think we should either move the quotes onto the same lines as the data string start/end or call `trim()` on the value before returning it.